Makefile

```diff
@@ -27,7 +27,9 @@ COCKPIT_REPO_STAMP=pkg/lib/cockpit-po-plugin.js
 # stamp file to check if/when npm install ran
 NODE_MODULES_TEST=package-lock.json
 APPSTREAMFILE=org.cockpit-project.$(PACKAGE_NAME).metainfo.xml
+ifeq ($(TEST_OS),)
 TEST_OS=fedora-rawhide-boot
+endif
```

Comment:
Shouldn't we use rather 
```
TEST_OS ?= fedora-rawhide-boot
```

Reply:
It's the same result, feel free to adjust if you prefer.